x/vm/migrations/v6/migrate.go

```diff
@@ -0,0 +1,49 @@
+package v6
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	legacytypes "github.com/cosmos/evm/legacy/types"
+	evmtypes "github.com/cosmos/evm/x/vm/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type evmKeeper interface {
+	SetCodeHash(ctx sdk.Context, addrBytes, hashBytes []byte)
+	GetParams(ctx sdk.Context) (params evmtypes.Params)
+	SetParams(ctx sdk.Context, params evmtypes.Params) error
+}
+
+func MigrateStore(ctx sdk.Context, ek evmKeeper, ak evmtypes.AccountKeeper) error {
```

Comment:
Could we add some unit tests for this function?

Reply:
added but since there is not testutils that set up test app, network, and keepers i had to add them under integration package

content of test is taken from evmos repo upgrade tests for this migration + i added part about params

i think this should be fine, even though location of test is not ideal, it will be removed in next release anyways